### ISSUE TITLE
Nice project

### ISSUE THREAD

**Issue by meta-gravity** [pertinate/fullstack-template#1]
But a question is there going to be a cli for it. Love what you are doing

**Comment by pertinate** [comment]
I wasn't initially planning for a cli. It was basically a template repo for me to reuse (for everyone to reuse as well). I kind of was toying with the idea of making it more module friendly? Not sure if that's the correct wording. I'd have to look up how to set up an npx command for it

**Comment by meta-gravity** [comment]
Yes that will be so nice and so easy.. I will be here waiting for that

**Comment by meta-gravity** [comment]
oh no problem 🚀🚀🚀🚀🚀🚀😁
